sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/aad/webapi/AADJwtBearerTokenAuthenticationConverterTest.java

```diff
@@ -78,7 +82,7 @@ public void testNoArgumentsConstructorExtractScopeAuthorities() {
 
     @Test
     public void testNoArgumentsConstructorExtractRoleAuthorities() {
-        when(jwt.containsClaim("roles")).thenReturn(true);
+        when(jwt.getClaim("scp")).thenReturn(null);
```

Comment:
Can we delete this line?

Reply:
Yes, if we delete this line, then we should update the assert line. Because the `jwt.getClaim("xxx")` is used in `Collection<GrantedAuthority> convert(Jwt jwt)`, the claim name sub and roles have been mocked in @BeforeEach method.
```
claimToAuthorityPrefixMap.forEach((authoritiesClaimName, authorityPrefix) -> {
            Optional.of(authoritiesClaimName)
                    .map(jwt::getClaim)
                    .map(this::getClaimValueAsCollection)
                    .map(Collection::stream)
                    .orElseGet(Stream::empty)
                    .map(authority -> authorityPrefix + authority)
                    .map(SimpleGrantedAuthority::new)
                    .forEach(grantedAuthorities::add);

        });
```

Reply:
It's OK to update the assert line. Just make sure the result is what we expected.